chapters/constants.md

```diff
@@ -16,12 +16,12 @@
 
 |  Parameter  				|  Value  		|  Description  																		|
 |--------------------------	|--------------	|------------------------------------------	|
-|`SATOSHIS_PER_COIN`		|1e5			|Number of satoshis per Nimiq. 															|
```

Comment:
That's an actual constant in the code, you'd have to change `SATOSHIS_PER_COIN` in the code to `LUNAS_PER_COIN` too.

Reply:
Whoops can't believe I missed that, I'm resubmitting the pull request with some more work done.